cmd/managementd/main.go

```diff
@@ -55,12 +55,14 @@ func main() {
 	router.HandleFunc("/camera-positioning", managementinterface.CameraPositioningHandler).Methods("GET")
 	router.HandleFunc("/network-interfaces", managementinterface.NetworkInterfacesHandler).Methods("GET")
 	router.HandleFunc("/disk-memory", managementinterface.DiskMemoryHandler).Methods("GET")
+	router.HandleFunc("/snapshot-image", managementinterface.GetSnapshot).Methods("GET")
 
 	// API
 	apiObj := api.NewAPI(cptvDir)
 	router.HandleFunc("/api/recordings", apiObj.GetRecordings).Methods("GET")
 	router.HandleFunc("/api/recording/{id}", apiObj.GetRecording).Methods("GET")
 	router.HandleFunc("/api/recording/{id}", apiObj.DeleteRecording).Methods("DELETE")
+	router.HandleFunc("/api/take-snapshot", apiObj.TakeSnapshot).Methods("GET")
```

Comment:
Let's make this a `PUT` to `/api/camera/snapshot`. A `GET` isn't supposed to change the server's state.